files/image_config/interfaces/init_interfaces

```diff
@@ -5,6 +5,4 @@
 auto lo
 iface lo inet loopback
 #
-# The management network interface
-auto eth0
-iface eth0 inet dhcp
+# Disable the management network interface during boot up
```

Comment:
What is the purpose of init_interfaces now? Can it be removed completely?

Reply:
Nikos and I discussed offline. We believe that if update_graph service is not enabled at build time. The interfaces-config service will either configure static IP or enable dhcp on eth0.

Nikos will run a quick test and get back.